src/backend/distributed/utils/maintenanced.c

```diff
@@ -139,6 +140,49 @@ InitializeMaintenanceDaemon(void)
 }
 
 
+/*
+ * InitializeMaintenanceDaemonForMainDb is called in _PG_Init
+ * at which stage we are not in a transaction or have databaseOid
+ */
+void
+InitializeMaintenanceDaemonForMainDb(void)
+{
+	if (strcmp(MainDb, "") == 0)
+	{
+		elog(LOG, "There is no designated main database.");
+		return;
+	}
+
+	BackgroundWorker worker;
+
+	memset(&worker, 0, sizeof(worker));
+
+	SafeSnprintf(worker.bgw_name, sizeof(worker.bgw_name),
+				 "Citus Maintenance Daemon: %u/%u",
+				 0, 0);
```

Comment:
I think instead of 0/0 it's more useful to users to write out what it is for.
```suggestion
	strcpy_s(worker.bgw_name, sizeof(worker.bgw_name),
				"Citus Maintenance Daemon for Main DB");
```